spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'Get /posts' do
+    it 'renders index template with correct placeholder text for posts' do
+      get '/posts'
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('List of All Posts')
+    end
+  end
+
+  describe 'Get /posts/:id' do
+    it 'renders show templates' do
+      get '/posts/id'
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:show)
+      expect(response.body).to include('Posts Details')
+    end
+  end
```

Comment:
- [ ] Great job adding your tests, however kindly ensure that they are all passing.

screenshot
![image](https://github.com/umair-rao/New-Blog-App/assets/81932478/02f968cc-e108-4446-88da-20113a980676)

Reply:
- Well resolved
<img width="514" alt="Screenshot 2024-01-20 at 09 11 05" src="https://github.com/umair-rao/New-Blog-App/assets/58692590/5268b7f6-ae73-46bb-85f8-8180eca9e82c">